styles/stylesheet.css

```diff
@@ -0,0 +1,566 @@
+* {
+  box-sizing: border-box;
+}
+
+.header {
+  height: 400px;
+  width: 100%;
+  background: linear-gradient(5deg, #9bc344, darkBlue);
+  background-position: top right;
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-image: url('../images/header-image.jpg');
+  overflow: hidden;
+}
+
+.header-checkout {
+  text-align: left;
+  position: absolute;
+  top: 10%;
+  right: 10%;
+  float: right;
+  clear: both;
+  font-family: 'Public Sans', sans-serif;
+  font-weight: 700;
+  font-size: .7em;
+  text-decoration: none;
+  color: #fff;
+  width: 10%;
+}
+
+.header-checkout a:link {
+  text-decoration: none;
+  text-align: center;
+  display: block;
+  padding: 0.3rem 0.8rem 0.3rem 0.8rem;
+  background-color: #9bc344;
+  border-radius: 1px;
+  border: none;
+  color: #fff;
+}
+
+.header-checkout a:visited {
+  color: #fff;
+  background-color: #9bc344;
+}
+
+.header-checkout a:hover {
+  color: #fff;
+  background-color: #6a872c;
+}
+
+.header-checkout a:active {
+  color: #fff;
+  background-color: #103861;
+}
+
+nav {
+  background-color: #ccc;
+  margin-top: -1rem;
+}
+
+ul {
+  list-style-type: none;
+  padding: 8px 0px 8px 0px;
+  margin-left: 3rem;
+}
+
+li {
+  display: inline;
+  padding-right: 0.8rem;
+}
+
+li a:link {
+  font-family: 'Public Sans', sans-serif;
+  font-weight: 700;
+  font-size: 0.8em;
+  text-decoration: none;
+  color: #103861;
+}
+
+li a:visited {
+  color: #000;
+}
+
+li a:hover {
+  color: #2480db;
+}
+
+li a:active {
+  color: #000;
+}
+
+.container-home {
+  margin: 3rem 0 0 3rem;
+}
+
+.container {
+  margin: 3rem 0 0 3rem;
+}
+
+h1 {
+  font-family: 'Merriweather', serif;
+  font-weight: 300;
+  font-style: italic;
+  font-size: 1em;
+  color: #9bc344;
+}
+
+h2 {
+  font-family: 'Merriweather', serif;
+  font-weight: 700;
+  font-size: 1.6em;
+  color: #103861;
+}
+
+h3 {
+  font-family: 'Merriweather', serif;
+  font-weight: 700;
+  font-size: 1.1em;
+  color: #103861;
+}
+
+h4 {
+  font-family: 'Merriweather', serif;
+  font-weight: 700;
+  font-size: 0.7em;
+  color: #103861;
+}
+
+h5 {
+  font-family: 'Merriweather', serif;
+  font-weight: 700;
+  font-size: 0.7em;
+  color: #103861;
+}
+
+p {
+  font-family: 'Public Sans', sans-serif;
+  font-weight: 300;
+  font-size: 0.8em;
+  line-height: 1.2em;
+}
+
+.intro-container {
+  display: block;
+  margin: 1.2rem 0.8rem 4rem 0rem;
+}
+
+.intro-container img {
+  max-width: 100%;
+}
+
+.intro-container h2 p {
+  float: right;
+  padding: 0.8rem 0.8rem 3rem 0;
+}
+
+.dog-profiles {
+  display: flex;
+  flex-flow: row wrap;
+  justify-content: start;
+  padding-top: 2rem;
+}
+
+.dog-profiles section {
+  border: 1px solid #ccc;
+  width: 25%;
+  margin: 0.8rem 3rem 3rem 0rem;
```

Comment:
Setting left and right margins to auto at smaller resolutions should center the dog tiles.